modules/i3s/src/lib/parsers/parse-i3s-webscene.ts

```diff
@@ -0,0 +1,46 @@
+import type {I3SWebScene, I3SWebSceneOperationalLayer} from '../../types';
+
+const ARCGIS_SCENE_SERVER_LAYER_TYPE = 'ArcGISSceneServiceLayer';
+
+/**
+ * Parses I3S WebScene and creates tileset
+ * @param data
+ * @param options
+ * @param context
+ */
+export async function parseI3SWebscene(data: ArrayBuffer, url: string): Promise<I3SWebScene> {
+  const layer0 = JSON.parse(new TextDecoder().decode(data));
+  const {operationalLayers} = layer0;
+
+  return {
+    header: layer0,
+    layers: parseOperationalLayers(operationalLayers)
+  };
+}
+
+/**
+ * Recursively parses WebScene Layer operational layers.
+ * @param sublayers
+ * @param url
+ */
+function parseOperationalLayers(
+  layersList: I3SWebSceneOperationalLayer[]
+): I3SWebSceneOperationalLayer[] {
+  let layers: I3SWebSceneOperationalLayer[] = [];
+
+  for (let index = 0; index < layersList.length; index++) {
+    const layer = layersList[index];
+    // Add support only for scene service layer for now.
+    // TODO Add ArcGISFeatureLayer support.
```

Comment:
This is a requirement. I'm not sure we need some special todo here. May be just comment that @loaders.gl supports only 3D scene layers.

Reply:
Change logic